frontend/src/store/auth-context.js

```diff
@@ -27,11 +28,17 @@ export const AuthContextProvider = (props) => {
     localStorage.setItem("token", token);
   };
 
+  const registerHandler = (token) => {
```

Comment:
this is doing exactly the same thing as `loginHandler`. perhaps re-use `loginHandler` instead of creating a new function?